DuckDuckGo/Updates/ReleaseNotesTabExtension.swift

```diff
@@ -198,7 +199,7 @@ private extension UpdateCycleProgress {
         case .downloadDidStart, .downloading: return .updateDownloading
         case .extractionDidStart, .extracting, .readyToInstallAndRelaunch, .installationDidStart, .installing: return .updatePreparing
         case .updaterError: return .updateError
-        case .updateCycleNotStarted, .updateCycleDone: return .updateReady
+        case .updateCycleNotStarted, .updateCycleDone: return .loaded
```

Comment:
These should've been defaulted to `loaded` as we only want `updateReady` if we truly have a pending update.